app/src/test/java/com/greenmonkeys/handmade/USDollarWalletTest.java

```diff
@@ -0,0 +1,49 @@
+package com.greenmonkeys.handmade;
+
+import com.greenmonkeys.handmade.payoutdenominations.USDollarWallet;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class USDollarWalletTest {
+    @Test public void testAdd() {
```

Comment:
This testAdd() doesn't add anything. Maybe update the test so that it is testing the add functionality or change the name of the test.